### ISSUE TITLE
Free trial: Frontend menu and banner missing when not proxied

### ISSUE THREAD

**Issue by ilyasfoo** [Automattic/wc-calypso-bridge#1075]
### Description

When not proxied, the top bar menu is pushed above screen

Proxied:

<img src="https://user-images.githubusercontent.com/3747241/231812231-41c13616-0e3b-4157-8e2e-bf99c3cbe7e1.png" width="600">

Not proxied:

<img src="https://user-images.githubusercontent.com/3747241/231812425-e6a5dac1-c58b-437c-9d66-ded716e3f116.png" width="600">

### To reproduce

1. Enable proxy
1. In your free trial site, go to shopper facing page
2. Observe that the top menu is displayed
3. Disable proxy
4. Refresh and observe that menu is no longer displayed along with the free trial banner

**Comment by chihsuan** [comment]
Hey @ilyasfoo 👋  I couldn't reproduce it with the instructions. 

Here is the screenshot on shopper facing page of my site without proxy:

![Screenshot 2023-04-27 at 14 34 18](https://user-images.githubusercontent.com/4344253/234779271-6cd6d681-6771-4345-8f8d-5f97f7544bdc.png)

I noticed that on your screenshot, the path is `/my-account-2`, but it's 404 pages on my site. Is that related to the theme? 

![Screenshot 2023-04-27 at 14 36 34](https://user-images.githubusercontent.com/4344253/234779592-099e50f3-6dd0-42dc-b69b-c891240c8817.png)

**Comment by ilyasfoo** [comment]
Apologies for the delayed response, I'll check again and get back to you @chihsuan

**Comment by ilyasfoo** [comment]
Hi, @chihsuan. I've tried this again in both my atomic development site and a fresh WooExpress site from NUX. I still can reproduce it.

I've tried it in Tsubaki theme and Blank canvas theme


https://user-images.githubusercontent.com/3747241/235075336-bee868d1-1c23-44ed-8956-73839aaf594d.mp4

**Comment by chihsuan** [comment]
Hey @ilyasfoo Interesting! I tried with a fresh WooExpress site from NUX (also using Brave), but I still can't reproduce it. 😅 

However, I noticed there is a wp admin bar there. I think that's the main difference. Do you have any clue?

![Screenshot 2023-04-28 at 15 21 25](https://user-images.githubusercontent.com/4344253/235081894-12f244c3-6cf7-4672-9e29-d841a099f423.png)

**Comment by ilyasfoo** [comment]
@chihsuan I remember the debug bar has always shown whenever I'm connected to proxy.

I did another test using a non-a8c account, it seems the issue went away:

<img width="1320" alt="image" src="https://user-images.githubusercontent.com/3747241/235100784-23e8071d-e295-4fe4-8fd1-5c9eb7fc6955.png">

Though interestingly debug bar is still shown even with non-a8c account:

<img width="1320" alt="image" src="https://user-images.githubusercontent.com/3747241/235100837-2502b187-217f-4448-8f1c-6ae59f710e15.png">

But since this wouldn't affect normal users, I think this issue is moot. Closing, thanks for helping me investigate this issue @chihsuan!
